subsys/bluetooth/host/Kconfig

```diff
@@ -112,7 +112,6 @@ config BT_RX_STACK_SIZE
 	default 3092 if BT_MESH_GATT_CLIENT
 	default 2600 if BT_MESH
 	default 2048 if BT_AUDIO
```

Comment:
Assuming BT_AUDIO enabled, does uses BT_SETTINGS and 2048 seemed to be sufficient?

Reply:
> Assuming BT_AUDIO enabled, does uses BT_SETTINGS and 2048 seemed to be sufficient?

I would assume so since that's what's used even before this PR: the defaults are evaluated from top down, and the first match gets used, i.e. the BT_AUDIO line always matched before the BT_SETTINGS one anyway.

Reply:
That said, since there was quite a lot of unnecessary overhead with the unconditional default, there's a good chance that there's wasted space in the conditional ones, i.e. it'd be good to do some new profiling there (@Thalley FYI).